FWCore/src/components/PodioOutput.cpp

```diff
@@ -128,5 +128,6 @@ StatusCode PodioOutput::finalize() {
   m_datatree->Write();
   m_file->Write();
   m_file->Close();
+  info() << "Data written to: " << m_filename << endmsg;
```

Comment:
Is this related to this PR?

Reply:
No. Do you think it needs its own PR?

Reply:
IMO it is better to have one single PR for logical change, but it's fine.